decisions/remote-server-backup.md

```diff
@@ -0,0 +1,57 @@
+# Remote Server Backup
+
+- *Date*: 2022-12-7
+- *Deciders*: 
+- *Status*: 
+- *Outcome*: 
+- *Related Material*: [Backup Issue](https://github.com/openmsupply/open-msupply/issues/767)
+
+## Context
+
+It's important to back up remote server data so that it can be restored in case of corruption/loss. This KDD is to discuss remote synchronised server backup.
+
+In existing 4d mSupply, for remote servers, we have backup files and current journal file. There is no way to re-initialise remote desktop mSupply sites, thus we must rely on local/dropbox backups. There have been cases with quite catastrophic outcomes where backup was restored incorrectly, causing major data synchronisation issues.
+
+### Wrongly Restored
+* Wrong backup (not the latest)
+* Forget to integrate journal
+* Journal data corruption
+
+### Major data syncrhonisation issues
+
+When remote data is not `in sync` with central data (i.e. older backup was used), we almost certainly have a ledge discrepancy that can persist and is hard to fix correctly. 
```

Comment:
:nit couple of spelling nits - the `ledge` here  and in the example and a `scenarior` later on. And possibly `disk` instead of `disc`